index.html

```diff
@@ -1,79 +1,86 @@
+      <ol class="breadcrumb mx-2">
+        <li class="breadcrumb-item"><a class="text-success" href="index.html">Accueil</a></li>
+        <li class="breadcrumb-item active" aria-current="page">À propos</li>
+      </ol>
+    </nav>
+    <h1 class="text-success text-center mt-5 mb-5 fs-3">Accueil</h1>
+    <div class="card m-5" style="max-width: 100%;">
+      <div class="row g-0">
+        <div class="col-md-4">
+          <img src="https://picsum.photos/400/200" class="img-fluid rounded-start" alt="Profile picture">
+        </div>
+        <div class="col-md-8">
+          <div class="card-body">
+            <h5 class="card-title mt-4">Résumé</h5>
+            <p class="card-text mt-3">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Possimus dolorem
+              reiciendis quibusdam aliquid ducimus facilis totam nam id ex quam aperiam nesciunt commodi amet,
+              voluptatibus, vitae ad facere dolore laudantium veritatis. Eos reiciendis odio accusamus repudiandae, sit
+              excepturi quae facilis.</p>
+          </div>
+        </div>
+      </div>
     </div>
-    <div class="container">
-        <div class="row">
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/dog.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
-            </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/woman.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
-            </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/man.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
-                </div>
-            </div>
+    <div id="carouselExample" class="carousel slide m-5">
```

Comment:
Sometimes your id's are camelCase and others kebab-case. Try to follow a standard.

Reply:
All id's are camelCase, and all style/class are kebab-case, they're all from the documentation of Bootstrap, I did not create any id myself.  Id "carouselExemple" cannot be modified else the bootstrap code for the carousel image slide will not work anymore.